actions/traits/LtiModuleTrait.php

```diff
@@ -39,9 +39,6 @@
      */
     protected function returnLtiError(\taoLti_models_classes_LtiException $error, $returnLink = true)
     {
-        // full trace of the error
-        \common_Logger::e($error->__toString());
```

Comment:
Agreed, we need to track these user errors, but should not do this as system errors.
Will come up with an alternative.